examples/operator/operator.yaml

```diff
@@ -74,8 +74,35 @@ spec:
                         - kind
                         - name
                       type: object
+                      x-kubernetes-map-type: atomic
```

Comment:
for my knowledge.. Is it some auto generated file ? and what are all these changes?

Reply:
It is auto generated, but on vitess-operator. Whenever it changes, we have to copy them here, but I forgot to do that this time. 😭 
Anyways these changes came from upgrading the controller-gen version from `controller-gen.kubebuilder.io/version: v0.6.2` to `controller-gen.kubebuilder.io/version: v0.11.3`